gui/slick/views/home_massAddTable.mako

```diff
@@ -12,21 +12,23 @@
             continue
 
         show_id = curDir['dir']
-        if curDir['existing_info'][0]:
-            show_id = show_id + '|' + str(curDir['existing_info'][0]) + '|' + str(curDir['existing_info'][1])
-            indexer = curDir['existing_info'][2]
+        if curDir['existing_info']['indexer_id']:
+            show_id = show_id + '|' + str(curDir['existing_info']['indexer_id']) + '|' + str(curDir['existing_info']['show_name'])
+            indexer = curDir['existing_info']['indexer']
 
         indexer = 0
-        if curDir['existing_info'][0]:
-            indexer = curDir['existing_info'][2]
+        if curDir['existing_info']['indexer_id']:
+            indexer = curDir['existing_info']['indexer']
         elif sickbeard.INDEXER_DEFAULT > 0:
             indexer = sickbeard.INDEXER_DEFAULT
     %>
```

Comment:
This entire python part is not needed anymore, but kept it for now. These are now replaced by the data attributes. But need to test if there arent' any issues. If working okay, i'll remove it.